src/modules/seventv/utils.js

```diff
@@ -1,11 +1,11 @@
 import {hasFlag} from '../../utils/flags.js';
 import Emote from '../emotes/emote.js';
 
-function emoteUrl(id, version, static_ = false) {
-  return `https://cdn.7tv.app/emote/${encodeURIComponent(id)}/${version}${static_ ? '_static' : ''}.webp`;
+function emoteUrl(url, version, static_ = false) {
+  return `https:${url}/${version}${static_ ? '_static' : ''}.webp`;
```

Comment:
is there any guarantee they won't add a scheme in front of the url? trailing slash?